### ISSUE TITLE
To create login & about page

### ISSUE THREAD

**Issue by aishwaryas15** [sifa123/Dev-Kitchen#3]

**Comment by sifa123** [comment]
Hi @aishwaryas15, Thanks for raising this issue. I believe you are one of our contributors from DevIncept, and you know it's going to start on 10th July, and so within some days they can make us a team, there I will give you some info on this project.

**Comment by aishwaryas15** [comment]
> Hi @aishwaryas15, Thanks for raising this issue. I believe you are one of our contributors from DevIncept, and you know it's going to start on 10th July, and within some days they could make us a team, there I will give you some info on this project.

Yeah, I'm from DevIncept. Ya, sure I'll wait till further notice.

**Comment by Shivani207-tech** [comment]
I would like to work on this issue. Please assign it to me under DCP 2021. Thankyou.

**Comment by sifa123** [comment]
@aishwaryas15 you can start working on that, you will find our channel link in random channel

**Comment by aishwaryas15** [comment]
Channel link?

**Comment by sifa123** [comment]
> Channel link?

Please DM me on slack

**Comment by sifa123** [comment]
@aishwaryas15 Please start designing the about page.

**Comment by aditya687** [comment]
I want to try this issue @sifa123
